app/gcl/professional-cloud-architect/section2-managing-provisioning/page.css

```
@@ -0,0 +1,619 @@
+    padding: 12px 16px;
+    font-weight: 700;
+    border-bottom: 1px solid var(--pca-s2-border);
+    white-space: nowrap;
+}
+
+.pca-s2-page tbody td {
+    padding: 12px 16px;
+    border-bottom: 1px solid var(--pca-s2-border-soft);
+    color: var(--pca-s2-text-dim);
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)